terraform/workspace-variables/review.tfvars

```diff
@@ -9,6 +9,6 @@ paas_app_start_timeout = "180"
 paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
-paas_web_app_memory = 1024
+paas_web_app_memory = 4096
```

Comment:
are we able to revert this before we merge?

Reply:
will need this to seed BNP data in m2-release, but will clean this up once we've got the ✅